admin/class-admin-apple-index-page.php

```diff
@@ -335,17 +311,6 @@ public function setup_assets( $hook ) {
 		);
 	}
 
-	/**
-	 * Shows the list of articles available for publishing to Apple News.
-	 *
-	 * @access public
-	 */
-	public function show_post_list_action() {
```

Comment:
just double checking that this func isn't used anywhere else.

Reply:
It's not - this class was the only place that referenced it. Rather than having the action hook call one function that just called this one, I opted to collapse the content of this function into the action callback. I suppose client code could have referenced this function directly because it's public, I'm not concerned about making this change in the name of housekeeping.